### PR TITLE
emnlp-ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Review by mjpost — COMMENTED** [review]
Title of main book should be "Proceedings of the..." (the currently missing).

File: data/xml/2021.codicrac.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (mjpost) [inline data/xml/2021.codicrac.xml:1]: `codicrac` should be not be recorded as a new venue, but as a volume within `codi` or `crac`, and then cross-listed within the other. I suggest you use `2021.codi-sharedtask` as the volume name, and then add that volume under `crac` in `data/venues/joint.yaml`.

File: data/xml/2021.latechclfl.xml [hunk 2]
```diff
@@ -0,0 +1,241 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (mjpost) [inline data/xml/2021.latechclfl.xml:1]: Same here. Don’t create a new venue. Use `2021.latech-1` as the volume name, then cross-list with `clfl`.

File: data/xml/2021.lawdmr.xml [hunk 3]
```diff
@@ -0,0 +1,209 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (mjpost) [inline data/xml/2021.lawdmr.xml:1]: Venue is `law`, volume is `2021.law-1`, listed as joint with `dmr`.

**Review by mjpost — COMMENTED** [review]
This is coming along, thanks for all the changes. A few left that I can see:

* [x] The main volume name of EMNLP is missing a “the” (see [the top of the page here](https://preview.aclanthology.org/emnlp-ingestion/events/emnlp-2021/))
* [ ] This “the” is also missing in the footer of the paper. Really they should regenerate it
* [ ] I had originally said that `latechclfl` should be split into `latech` and `clfl`, but it looks like they’ve used this name for the past two years, and perhaps merged in 2020? All three venues exist: [clfl](https://preview.aclanthology.org/emnlp-ingestion/venues/clfl/), [latech](https://preview.aclanthology.org/emnlp-ingestion/venues/latech), and [latechclfl](https://preview.aclanthology.org/emnlp-ingestion/venues/latechclfl). I guess we should just keep it.
* [x] `lawdmr` needs to be split into two, file under `law`, co-list with `dmr`
* [ ] We do need edits to `joint.yaml`:
   - to co-list joint venues mentioned earlier
   - to add all workshops to the EMNLP page

File: data/xml/2021.lawdmr.xml (hunk 3 above)
Comment (mjpost) [reply data/xml/2021.lawdmr.xml]: Done.

File: data/xml/2021.latechclfl.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2021.latechclfl.xml:1]: Here, it seems the venues have merged. So forget this comment.

File: data/xml/2021.codicrac.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2021.codicrac.xml:1]: Done.

**Comment by xinru1414** [comment]
@mjpost Please take a look at `joint.yaml` re how `latechclfl`, `latech` and `clfl` are co-listed

**Review by mjpost — APPROVED** [review]
LGTM. Thanks @xinru1414 for your extensive work on this!

**Pull request merged**